buildtools/webpack/plugins.js

```diff
@@ -0,0 +1,80 @@
+/* global module module */
+/*eslint camelcase: ["error", {properties: "never"}]*/
+
+var fs      = require('fs');
+var webpack = require('webpack');
+var _       = require('underscore');
+
+var license = fs.readFileSync('src/widget/copyright.txt', 'utf8');
```

Comment:
This is somewhat ambiguous because it depends on where this is being run (root dir) - nicer to be explicit i.e.

```javascript
const { join } = require('path');

const licence = readFileSync(join(__dirname, '../../src/widget/copyright.txt'), 'utf8');
```